htmresearch/algorithms/location_modules.py

```diff
@@ -184,6 +200,10 @@ def activateRandomLocation(self):
     Set the location to a random point.
     """
     self.activePhases = np.array([np.random.random(2)])
+    if self.anchoringMethod == "discrete":
+      # Need to place the phase in the middle of a cell
+      self.active_phases = np.floor(
+      self.activePhases * self.cellDimensions)/self.cellDimensions
```

Comment:
indentation is off here

also you have `self.active_phases` being set using `self.activePhases`. Is that a typo? Regardless, use the camelcase version and change the name of the other to clearly distinguish it if they aren't supposed to be the same